common/pulp/common/constants.py

```diff
@@ -59,13 +59,13 @@
 # Constant used to determine whether a CeleryBeatLock should be removed due to age
 CELERYBEAT_LOCK_MAX_AGE = 200
 # The amount of time in seconds before a Celery process is considered missing
-CELERY_TIMEOUT_SECONDS = 300
+CELERY_TIMEOUT_SECONDS = 60
```

Comment:
It looks like this change, along with the one below for ``CELERY_MAX_INTERVAL``, are already in a separate commit. It does also have a release note, although elaborating on its impact may be beneficial.

I think this is a good change for the 2.y line.